share/functions/__fish_anypython.fish

```diff
@@ -3,7 +3,7 @@ function __fish_anypython
     # Try python3 first, because that's usually faster and generally nicer.
     # Do not consider the stub /usr/bin/python3 that comes installed on Darwin to be Python
     # unless Xcode reports a real directory path.
-    for py in python3 python3.{12,11,10,9,8,7,6,5,4,3} python2 python2.7 python
+    for py in python3 python3.{14,13,12,11,10,9,8,7,6,5}
```

Comment:
thanks, queueing.

If you want to do more work on this,
I think we should get rid of the fallbacks to minor versions,
I think it only exists for NetBSD;
for that we can add a CMake flag (`-DPYTHON_EXECUTABLE`),
which is probably also useful to NixOS folks